webpack.config.js

```diff
@@ -83,10 +83,14 @@ module.exports = [
 			filename: 'jp-search.bundle.js',
 		},
 		performance: isDevelopment
-			? {}
+			? {
+					maxAssetSize: 120000,
```

Comment:
Shouldn't we keep smaller limits in development, so we get blocked in production instead of only seeing the problems on release day when trying the production build?

Reply:
The problem is that the dev size is actually closer to 110 Kb. I don't know the exact reason behind that, probably it's due to dev environment, hence no minimization and no uglifying? If we lower that any more than this, dev builds will start to fail.